### PR TITLE
globjects 1.0.0 (new formula)

### PR DESCRIPTION
**Pull request by andrew-kennedy**
just adding the globjects library to homebrew.

### CODE REVIEW

File: Formula/globjects.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  head "https://github.com/cginternals/globjects.git"
+
+  depends_on "cmake" => :build
+  depends_on "glbinding"
+
+  needs :cxx11
+  def install
+    args = []
+
+    args << "-Dglbinding_DIR=" + Formula["glbinding"].prefix
```
Comment (bfontaine) [inline Formula/globjects.rb]: You don’t need to build an array here; just add `"-Dglbinding_DIR=#{Formula["glbinding"].opt_prefix}"` to the arguments list below.

File: Formula/globjects.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  desc "C++ library strictly wrapping OpenGL objects"
+  homepage "https://github.com/cginternals/globjects"
+  url "https://github.com/cginternals/globjects/archive/v1.0.0.tar.gz"
+  sha256 "be2f95b4e98eef61a57925985735af266fef667eec63a39f65def5d5d808a30a"
+  head "https://github.com/cginternals/globjects.git"
+
+  depends_on "cmake" => :build
+  depends_on "glbinding"
+
+  needs :cxx11
```
Comment (bfontaine) [inline Formula/globjects.rb]: Please leave an empty line after that one.

**Review by bfontaine — CHANGES_REQUESTED** [review]

**Review by bfontaine — APPROVED** [review]

**Comment by andrew-kennedy** [comment]
Tests are succeeding on my machine but failing on TestBot build, probably due to glm dependencies auto resolving. Will try to modify test cases to correctly resolve the includes

**Comment by MikeMcQuaid** [comment]
Merged in a9228978237503e118fc1d730a7e221ab147c0a4.

Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**